app/src/utils/session.py

```diff
@@ -13,3 +15,5 @@ def init_session_state():
     st.session_state["realization"] = None
     st.session_state["block"] = None
     st.session_state["search_id"] = None
+    st.session_state["stac_api"] = os.getenv("STAC_API_URL")
+    st.session_state["stac_browser"] = os.getenv("STAC_BROWSER_URL")
```

Comment:
Can we call these `"stac_api_url"` and `"stac_browser_url"` for consistency?

Reply:
Yep!